### PR TITLE
Add basic styles for markdown components

### LINKED ISSUE #220: "Strong" text generated by markdown is not rendered with bold styling
**Before we start...:**

- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

**Branch/Commit:**

`main`

**Expected behavior:**

If I wrap text in double asterisks (`**like this**`), the preview and story should render the text with a bold style.

**Actual behavior:**
Wrapping the text with double asterisks renders a `<strong>` tag, but the style is overridden by a CSS reset and the text appears normal.

Please include a detailed description of the actual behavior of the application.

**Steps to reproduce:**

1. Wrap text in double asterisks in the description while adding or editing a story.
2. Preview text is not bold.

**Screenshots and Videos**
Screen Shot 2022-02-23 at 6 08 02 PM

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

### LINKED ISSUE #221: Add `:hard_wrap: true` to markdown renderer options
**Before we start...:**

- [x] I checked the documentation and didn't find this feature
- [x] I checked to make sure that this feature has not already been requested

**Branch/Commit:**
 `main`

**Describe the feature:**

This would allow users to create extra line breaks (rendered as `<br>` tags) to aid in formatting stories.

If I understand the `redcarpet` docs correctly (linked below), with `:hard_wrap: true`, you can create an extra line break by adding two spaces after the line before hitting `enter`/`return`.

**Problem:**

The only way I found to render a space between lines of text is to use `&nbsp;`.
It's not the worst thing in the world, but when trying to format stories well, the entire text box can quickly fill up with them.

Screen Shot 2022-02-23 at 6 09 48 PM

(after i took this screen shot, i noticed that `filter_html: true` is set, which I think is why `<br>` didn't work).

Please include a detailed description of the problem this feature would solve.

> **As a developer writing a story description**
> **I want to easily add line breaks**
> **So that I can format my descriptions well**

**Resources:**
Section of `redcarpet` docs that details the option:
https://github.com/vmg/redcarpet#darling-i-packed-you-a-couple-renderers-for-lunch

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

### LINKED ISSUE #222: `li` tags render unstyled in description previews and stories
**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**

**Before we start...:**

- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

**Branch/Commit:**

`main`

**Expected behavior:**
List items (`<li>` tags) rendered in the preview and stories should be styled as list items.

**Actual behavior:**
List items are styled with `list-style: none` and no padding, so they look like normal text.

**Steps to reproduce:**

1. Create a list in the description textarea:
```
* an
* example
* markdown list
```
2. The rendered list has no list styles.

**Screenshots and Videos**

Screen Shot 2022-02-23 at 7 01 56 PM

(i also tested it using `-` instead of `*`)

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

### PR DESCRIPTION
**Pull request by mcelaney**
Closes #220

Closes #221 

Closes #222 

**Description:**

Redcarpet converts markdown to markup but our css reset removes all styling from the result.

Screen Shot 2022-03-10 at 3 04 17 PM

Added styles for italics, bold, and lists on the preview:

Screen Shot 2022-03-10 at 3 03 58 PM

...and on the story detail page:

Screen Shot 2022-03-10 at 3 03 39 PM

___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md).

jira link https://ombulabs.atlassian.net/browse/DT-215

### CODE REVIEW

File: app/assets/stylesheets/stories.scss
Comment (arielj) [inline app/assets/stylesheets/stories.scss:58]: there is already a `.modal .story-description p` selector defined at line 39, we can add the margin there, it already includes the `overflow: auto` property

File: app/assets/stylesheets/stories.scss
Comment (arielj) [inline app/assets/stylesheets/stories.scss:43]: maybe we can nest the `em`, `strong` and `ul` style the same way the `p` is nested inside `.story-description` in line 13, to remove some repetition

Comment (arielj) [inline app/assets/stylesheets/stories.scss:43]: maybe we can nest the `em`, `strong` and `ul` style the same way the `p` is nested inside `.story-description` in line 13, to remove some repetition

File: app/assets/stylesheets/stories.scss
Comment (arielj) [inline app/assets/stylesheets/stories.scss:105]: I'd say to move these styles to reuse the same style set for `.story-description` above, it's the same style, maybe doing `.story-description, .story_preview` in line 13.

Comment (arielj) [inline app/assets/stylesheets/stories.scss:105]: I'd say to move these styles to reuse the same style set for `.story-description` above, it's the same style, maybe doing `.story-description, .story_preview` in line 13.

**Review by arielj — COMMENTED** [review]
hey @mcelaney , I added some refactor suggestions

**Comment by mcelaney** [comment]
Hey @arielj I think I got all that - could you have another look?

**Comment by kindoflew** [comment]
hi again, @arielj! just a reminder about this PR also. thanks!

**Review by arielj — APPROVED** [review]
LGTM!

**Comment by rishijain** [comment]
I have QA this and it is working as expected.

**Comment by rishijain** [comment]
hey @mcelaney changes look good and QA also is good.
Can you resolve the merge conflicts and push the changes?

**Comment by bronzdoc** [comment]
We can merge this after the merge conflicts are solved, thanks @mcelaney!

**Comment by mcelaney** [comment]
@rishijain Should be good to go

**Comment by rishijain** [comment]
all looks good in QA. merging this in. thanks @mcelaney  :)

**Pull request merged**
